compiler/src/dmd/iasmgcc.d

```diff
@@ -406,7 +410,7 @@ unittest
     {
         const errors = global.errors;
         scope gas = new GccAsmStatement(Loc.initial, tokens);
-        scope p = new Parser!ASTCodegen(null, ";", false);
+        scope p = new Parser!ASTCodegen(null, ";", false, global.errorSink);
```

Comment:
At this point, might as well make it a default param instead, there are dozens of repetitions of this pattern of change.